web/fora.hoon

```diff
@@ -0,0 +1,18 @@
+/?  310
```

Comment:
All the fora files belong in a different PR. They must have been added before you added those paths in the .gitignore.

Reply:
A PR to [urbit/fora](https://github.com/urbit/fora), to be precise.

Reply:
removed it from this repo, PR to urbit/fora coming in a bit